pyro/contrib/gp/models/model.py

```diff
@@ -134,22 +136,33 @@ def set_data(self, X, y=None):
 
         Some examples to utilize this method are:
 
+        .. doctest::
+           :hide:
```

Comment:
does this mean everything in this indentation block is collapsible or not rendered?

Reply:
@jpchen - This won't be rendered by sphinx.